converter/internal/promtailconvert/testdata/unsupported.diags

```diff
@@ -10,7 +10,6 @@
 (Warning) server.log_level is not supported - Flow mode components may produce different logs
 (Error) server.http_path_prefix is not supported
 (Warning) server.health_check_target disabling is not supported in Flow mode
-(Error) DropRateLimitedBatches is currently not supported in Grafana Agent Flow
```

Comment:
Would be good to test a promtail to flow conversion, which can be done by modifying the `testdata` files.

Reply:
Added a test in https://github.com/grafana/agent/pull/4865/commits/89a101688c68c3354b531dddf1b8e69f05018f69. I did not add the auth tests because I would need to create several files to try the different types of auth and I think that it should be done in another PR (if we really need full coverage of the params)

Reply:
Sounds good!